charts/vh-admin-web/values.yaml

```diff
@@ -28,27 +28,52 @@ java:
         - kinlyconfiguration--conferencephonenumber
         - kinlyconfiguration--conferencephonenumberwelsh
         - kinlyconfiguration--joinbyphonefromdate
+        - name: FeatureToggles--SDK-Key
+          alias: FeatureToggles--SdkKey
+    vh-bookings-api:
+      excludeEnvironmentSuffix: false
+      resourceGroup: vh-infra-core-{{ .Values.global.environment }}
+      usePodIdentity: true
+      secrets:
+        - name: azuread--identifieruri
+          alias: VhServices--BookingsApiResourceId
+    vh-video-api:
+      excludeEnvironmentSuffix: false
+      resourceGroup: vh-infra-core-{{ .Values.global.environment }}
+      usePodIdentity: true
+      secrets:
+        - name: azuread--identifieruri
+          alias: VhServices--VideoApiResourceId
+    vh-notification-api:
+      excludeEnvironmentSuffix: false
+      resourceGroup: vh-infra-core-{{ .Values.global.environment }}
+      usePodIdentity: true
+      secrets:
+        - name: azuread--identifieruri
+          alias: VhServices--NotificationApiResourceId
+    vh-user-api:
+      excludeEnvironmentSuffix: false
+      resourceGroup: vh-infra-core-{{ .Values.global.environment }}
+      usePodIdentity: true
+      secrets:
+        - name: azuread--identifieruri
+          alias: VhServices--UserApiResourceId
   environment:
     ASPNETCORE_URLS: http://+:8080
     AZUREAD__BASEVIDEOURL: https://vhinfracore{{ lower .Values.global.environment }}.blob.core.windows.net/video
     AZUREAD__POSTLOGOUTREDIRECTURI: https://vh-admin-web.{{ .Values.global.environment }}.platform.hmcts.net/logout
     AZUREAD__REDIRECTURI: https://vh-admin-web.{{ .Values.global.environment }}.platform.hmcts.net/home
     DISABLEHTTPSREDIRECTION: true
-    FEATURETOGGLE__SDKKey: "PLACEHOLDER"
     ISLIVE: "False"
     Logging__LogLevel__Default: debug
     Logging__LogLevel__System: debug
     Logging__LogLevel__Microsoft: debug
     TESTUSERSECRETS__TESTUSERNAMESTEM: "@hearings.reform.hmcts.net"
     TESTUSERSECRETS__TESTUSERPASSWORD: "PLACEHOLDER"
-    VHSERVICES__BOOKINGSAPIRESOURCEID: https://vh-bookings-api.{{ .Values.global.environment }}.platform.hmcts.net
-    VHSERVICES__BOOKINGSAPIURL: https://vh-bookings-api.{{ .Values.global.environment }}.platform.hmcts.net/
     VHSERVICES__LAUNCHDARKLYCLIENTID: "PLACEHOLDER"
```

Comment:
are these not in the KVs? --referring to "PLACEHOLDER"

Reply:
latest change did not push up :)